### PR TITLE
#49 Clippy Fix 3

### PR DESCRIPTION
**Pull request by pzheng1025**
Ignore unused codes.
Skip to fix too many arguments.

### CODE REVIEW

File: async_fuse/src/fuse_request.rs [hunk 1]
```diff
@@ -826,4 +826,10 @@ mod tests {
             _ => panic!("unexpected request operation"),
         }
     }
+
+    #[test]
```
Comment (rogercloud) [inline async_fuse/src/fuse_request.rs:830]: What's this test for?

Comment (pzheng1025) [reply async_fuse/src/fuse_request.rs:830]: > What's this test for?
To fix this, it is easy to add a test case, so I don't use #[allow(dead_code)]
warning: method is never used: `len`
  --> async_fuse/src/fuse_request.rs:17:5
   |
17 |     pub fn len(&self) -> usize {
   |     ^^^^^^^^^^^^^^^

Comment (pwang7) [reply async_fuse/src/fuse_request.rs:830]: Will test remove clippy unused warning?

Comment (pzheng1025) [reply async_fuse/src/fuse_request.rs:830]: Just tested. It won't. Will submit a new one.

**Comment by pwang7** [comment]
Please update branch before merge

**Pull request merged**